pkg/apis/keycloak/v1alpha1/keycloakbackup_types.go

```diff
@@ -7,16 +7,53 @@ import (
 // KeycloakBackupSpec defines the desired state of KeycloakBackup
 // +k8s:openapi-gen=true
 type KeycloakBackupSpec struct {
-	Restore bool            `json:"restore,omitempty"`
-	AWS     KeycloakAWSSpec `json:"aws,omitempty"`
+	// Controls automatic restore behavior.
+	// Currently not implemented (yet)
+	// +optional
+	Restore bool `json:"restore,omitempty"`
+	// If provided, an automatic database backup will be created on AWS S3.
```

Comment:
Change wording to say backup will be done on AWS S3.

Reply:
Fixed.